test/servico/Cst24Test.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['import' should be separated from previous statement.](https://www.codacy.com/app/calebepb/substituicao/file/2769137718/issues/source?bid=3308597&fileBranchId=3353677#l15)